app/src/main/res/layout/activity_main.xml

```diff
@@ -265,4 +265,33 @@
             app:layout_constraintTop_toTopOf="@+id/lblGPSSpeed"
             app:layout_constraintVertical_bias="0.0" />
 
+        <Button
+            android:id="@+id/button"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:layout_marginTop="512dp"
+            android:onClick="exportTest"
+            android:text="ExportTest"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
```

Comment:
I don't know if you want a seperate ticket for this, but:
TODO: replace function in existing export button with FTP single upload function and remove old export code.